features/proxy_config.feature

```diff
@@ -124,208 +117,207 @@ Feature: Proxy configuration
            | bionic  |
            | focal   |
 
-    @slow
-    @series.xenial
-    @series.bionic
-    @uses.config.machine_type.lxd.vm
-    Scenario Outline: Attach command when proxy is configured
-        Given a `<release>` machine with ubuntu-advantage-tools installed
-        When I launch a `focal` `proxy` machine
-        And I run `apt install squid -y` `with sudo` on the `proxy` machine
-        And I add this text on `/etc/squid/squid.conf` on `proxy` above `http_access deny all`:
+        @slow
```

Comment:
It looks like everything below this point got indented an extra 4 spaces, which makes it inconsistent with the first scenario and makes the diff hard to read :/ so we should unindent this.